kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -41,10 +60,11 @@ You will need:
   - Maven 3.5.4+ installed
 
 When using native image compilation, you will also need: 
-  - GraalVM 19.1+ installed
+  - GraalVM 19.1+ installed [prerequisites] with quarkus 1.3.0.*, GraalVM 19.3.1 minimum needed
```

Comment:
as we are already using 1.3.0.Alpha2 in master, you can directly set 

`GraalVM 19.3+ installed`

Reply:
fixed @radtriste